editor.planx.uk/src/@planx/components/Feedback/components/FaceBox.tsx

```diff
@@ -20,36 +20,37 @@ export const FaceBox = ({
   value,
 }: FaceBoxProps): ReactElement => {
   return (
-    <Grid item xs={2} key={label}>
+    <Grid item xs={2.4} key={label}>
       <ToggleButton
         value={value}
         data-testid={testId}
         sx={{
           px: 0,
+          width: "100%",
+          textTransform: "none",
+          "&[aria-pressed='true'] > div": {
```

Comment:
```suggestion
          [`&.${toggleButtonClasses.selected}`]: {
            div: {
```

It's unlikely that MUI will change how they use `aria-pressed` but we're a bit safer relying on their classes to monitor state within a component in instances like this. 

Docs: https://mui.com/material-ui/api/toggle-button/#classes

Reply:
Thanks for this, I've updated